src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java

```diff
@@ -284,4 +292,200 @@ public void testChangeInConfig() throws Exception {
         assertEquals(1, detailsAfterChangeInConfig.size(), "Invalid cache size");
     }
 
+    /**
+     * This SuppressWarning("unchecked") required to suppress
+     * "Unchecked generics array creation for varargs parameter" during mock.
```

Comment:
comment above looks like either a leftover from some previous changes or detached, there is no annotations

Reply:
done